setupJest.js

```diff
@@ -10,17 +10,6 @@ expect.extend({
 });
 
 expect.extend({
-  toBeInInitialisingState(uid2) {
-    expect(uid2.getAdvertisingToken()).toBeUndefined();
-    expect(uid2.isLoginRequired()).toBeUndefined();
-
-    return {
-      pass: true,
-      message: () =>
-        'Expected getAdvertisingToken() returns undefined and isLoginRequired() returns undefined',
-    };
-  },
-
```

Comment:
This state was only used in one test, and isn't really relevant anymore since an advertising token can be found without init